2021-03-01 16:27:23 INFO logger.py:126 Log level: INFO
2021-03-01 16:27:23 INFO main.py:259 GNS3 GUI version 2.2.18
2021-03-01 16:27:23 INFO main.py:260 Copyright (c) 2007-2021 GNS3 Technologies Inc.
2021-03-01 16:27:23 INFO main.py:261 Application started with /sbin/gns3
2021-03-01 17:00:37 ERROR node.py:492 error while deleting ubuntu-1: Node ID f72512a1-79ab-47a2-867a-25f3a3018616 doesn't exist
2021-03-01 17:00:37 CRITICAL __init__.py:1626 Node ID f72512a1-79ab-47a2-867a-25f3a3018616 doesn't exist
2021-03-01 17:19:22 WARNING __init__.py:222 You have unsaved preferences in IOS on UNIX.

Continue without saving?
2021-03-01 17:21:52 INFO appliance_wizard.py:238 Image '/home/saad/Documents/demo_material/IOU/Cisco IOU L3/i86bi_linux-adventerprisek9-ms.155-2.T.bin' has been successfully uploaded
2021-03-01 17:24:16 INFO appliance_wizard.py:238 Image '/home/saad/Documents/demo_material/IOU/Cisco IOU L2/i86bi-linux-l2-adventerprisek9-15.2a.bin' has been successfully uploaded
2021-03-01 17:24:37 INFO appliance_wizard.py:238 Image '/home/saad/Documents/demo_material/IOU/Cisco IOU L2/i86bi-linux-l2-adventerprisek9-15.2a.bin' has been successfully uploaded
2021-03-01 17:34:09 WARNING link.py:229 Error while creating link: Connecting a ethernet interface to a serial interface is not allowed
2021-03-01 17:34:17 WARNING link.py:229 Error while creating link: Connecting a ethernet interface to a serial interface is not allowed
2021-03-01 17:36:17 CRITICAL __init__.py:1031 Invalid name detected for this node: Router 1
2021-03-01 17:36:27 CRITICAL __init__.py:1031 Invalid name detected for this node: Router 2
2021-03-01 17:40:26 WARNING __init__.py:222 You have unsaved preferences in General.

Continue without saving?
